src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,357 @@
+
+## Refactors
+
+### Removal of MindShield implants being orderable from Cargo
+
+MindShields are metagameable and make it extremely unfun to play as a Head Rev. They have no interesting interactions; they’re just a hand-wave for why some people are completely immune to a game mechanic, and whilst that’s utterly required for round flow (allowing Sec and the Heads to trust each other) it is a magic solve-everything button which leads to a total wall in design.
+
+A small number of MindShields could be kept on-station to allow staff promotions within reason. _A kind admin might decide to send a crate if the station runs out and it'd be good for the round, but this is out of scope of this design document._
+
+### Removal of Flashes being able to convert people
+
+Flashes are metagameable, confusing for newbies, hard to balance, and enforce too much reliance on manufacture by Science by the Revs for effective round flow. They also have no narrative justification beyond it being an Ur-mode from the dawn of SS13. There are people playing SS14 that are younger than the only good _Men In Black_ movie.
+
+### Revs win by stealing some Head ID cards and storming the bridge
+
+This removes one half of the round-removal problem and gives the Revs a clear final target that works a bit like the Nukie bomb.
+
+To compromise the terminal, the Revs need access to a Head Of Staff ID. Any will do: using them on the terminal allows for the Uplink to be disabled after a timer.
+
+Each ID used on the terminal reduces the amount of time the countdown will take by a certain amount.
+
+The start of the countdown is broadcast as a station announcement:
+
+`CENTRAL COMMAND UPLINK SEVERANCE PROCESS INITIALIZED BY [names on IDs here]. THIS STATION WILL BE REMOVED FROM NANOTRASEN SECURITY CONTROLS IN [amount of seconds]. IF THIS IS UNINTENTIONAL, PLEASE CANCEL THIS PROCESS BY VISITING THIS UPLINK TERMINAL AT CO-ORDINATES [co-ordinates here]`
+
+When this countdown reaches one minute, similar to the Nuke Ops game mode, a short theme plays.
+
+If the timer reaches zero, if it's a Rev round, this triggers victory. This totally shuts down station security, and all passes become All-Access. (This occurs even if the round is not Revs).
+
+If this is a Rev round, the Head Revolutionary passes are renamed "Consul" and all Head and Sec passes are renamed "Enemies of the Revolution". But this is just for flavour, as we're into EOR gib-fests at this point.
+
+_If this **isn't** a Revs round and this has happened, this would be an excellent time for CentComm to send an ERT._
+
+### Deconversion of Head Revolutionaries and Revolutionaries
+
+Head Revs are de-antagged by having their Revolutionary Implant removed.
+
+Revs are de-antagged by being exposed to Loyalty Loudspeakers for a bit of time.
+
+## New Content
+
+### Revolutionary Implant given to all Head Revolutionaries
+
+This implant does three things:
+
+1. Gives the HR innate access to talk on the Syndicate radio.
```

Comment:
Giving Private radio comms is strong in my opinion. Giving them private communication without a trade of sorts feels over the top. For traitors, they need to use their TC, a limited resource, to get it. and even then hope that other traitors (if they havent found any yet) also get it. Having it start for HeadRevs for essentially free, can make snowballing a lot easier.